Directory.Build.props

```diff
@@ -36,7 +36,10 @@
     <BuildArch Condition=" '$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)' == 'Arm64' ">arm64</BuildArch>
     <BuildArch Condition=" '$(BuildArch)' == '' ">amd64</BuildArch>
     <DcpDir>$(NuGetPackageRoot)microsoft.developercontrolplane.$(BuildOs)-$(BuildArch)/$(MicrosoftDeveloperControlPlanedarwinamd64PackageVersion)/tools/</DcpDir>
-    <AspireDashboardDir>$(DotNetOutputPath)Aspire.Dashboard/$(Configuration)/$(TargetFramework)/</AspireDashboardDir>
+
+    <!-- TODO: Need to figure out we can automatically detect target framework here. This property
+               is specified to support dashboard path metadata generation on the inner loop. -->
+    <AspireDashboardDir>$(MSBuildThisFileDirectory)/artifacts/bin/Aspire.Dashboard/$(Configuration)/net8.0/</AspireDashboardDir>
```

Comment:
This `Directory.Build.props` file will be running before the `.csproj` file. Due to evaluation order, the `TargetFramework` property here won't be defined.

Options are to move this to a `.targets` file, or to a target.

Reply:
Will tackle this in a follow up PR.